doc/man1/mac.pod

```diff
@@ -0,0 +1,151 @@
+=pod
+
+=head1 NAME
+
+openssl-mac,
+mac - perform Message Authentication Code operations
+
+=head1 SYNOPSIS
+
+B<openssl mac>
+B<-name <mac_name>>
+[B<-help>]
+[B<-ctrl>]
+[B<-bin>]
+[B<-out filename>]
+[B<file>]
+
+B<openssl> I<mac> B<-name <mac_name>>  [B<...>]
+
+=head1 DESCRIPTION
+
+The message authentication code functions output the MAC of a supplied file
+in hexadecimal.
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<-help>
+
+Print out a usage message.
+
+=item B<-name <mac_name>>
+
+Specifies name of a supported mac algorithm to be used. To see the list of
+supported mac's, use the command I<list -mac-algorithms>. This option must
+be specified.
+
+=item B<-bin>
+
+Output the MAC in binary form. Uses hexadecimal text format if not specified. 
+
+=item B<-out filename>
+
+Filename to output to, or standard output by default.
+
+=item B<-ctrl nm:v>
+
+Passes options to the MAC algorithm.
+The following options are supported:
+
+=over 4
+
+=item B<key:string>
+
+Specifies MAC key as alphanumeric string (use if key contain printable
+characters only). String length must conform to any restrictions of
+the MAC algorithm. A key must be specified for any MAC algorithm.
+
+=item B<hexkey:string>
+
+Specifies MAC key in hexadecimal form (two hex digits per byte).
+Key length must conform to any restrictions of the MAC algorithm.
```

Comment:
The key length must...

the _chosen_ MAC algorithm.  Or _selected_.

Reply:
Not needed - this is like this in multiple places